css-three-column-layout/css-three-column-layout/styles.css

```diff
@@ -0,0 +1,93 @@
+* {
+  background: #039be5;
+  color: white;
+  box-sizing: border-box;
+  min-width: 0;
+  min-height: 0;
+}
+
+body {
+  margin: auto;
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+header {
+  padding: 30px;
+  text-align: center;
+}
+
+img {
+  width: 82px;
+  height: 85px;
+  background-color: white;
+  align-content: center;
+}
+
+h3 {
+  font-size: 23px;
+  width: 300px;
+}
+
+p {
+  color: gainsboro;
+  width: 320px;
+  font-size: 18px;
+}
+
+.container {
+  max-width: 1200px;
+  margin: auto;
+}
+
+.row-1 {
```

Comment:
Rows should be percentages in width, typically 100% which is the default size for block level elements. Also, you only need 1 `row` class. If you need to add additional CSS, I would recommend using a different class so you don't pollute the `row` class.